command/server.go

```diff
@@ -113,22 +114,47 @@ func (c *ServerCommand) Run(args []string) int {
 		return 1
 	}
 
+	var advertiseAddr string = config.Backend.AdvertiseAddr
+
+	// Note that "habackend" is a backend that *may* support HA;
+	// it defaults to the same backend as normal operations
+	var habackend physical.Backend = backend
```

Comment:
Nitpick: let's just do `advertiseAddr := ...` here and `habackend := backend`. I don't think there is any benefit in explicitly stating the type since both are already the same type (sometimes you need to I know to generalize the interface but in this case I think it just works).